lib/index.js

```diff
@@ -94,7 +94,13 @@ async function getConfig(context, fileName, defaultConfig) {
   }
 
   const baseParams = getBaseParams(params, config[BASE_KEY]);
-  const baseConfig = await loadYaml(context, baseParams);
+  let baseConfig = await loadYaml(context, baseParams);
+  if (baseConfig == null && baseParams.repo === '.github') {
```

Comment:
If the repo is named `.github` *and* it is being used with `_extends`, I think it should skip looking in the `.github` directory entirely and only look at the root. Any opposition to that?

Reply:
If we are going to do that, I would suggest *always* looking at the root.

The only concern I have about not looking at the root is the fact that some projects (your own? I think you mentioned somewhere) already use a ".github" repo, and might be confused with an update to probot-config, but this isn't a major issue.

When it's looking at ".github" automatically without _extends, I think it make sense to only support looking at the root, since there's no precedent set for that at this point.